R/pkg/tests/fulltests/test_sparkSQL.R

```diff
@@ -4098,14 +4098,13 @@ test_that("catalog APIs, listTables, getTable, listColumns, listFunctions, funct
                c("name", "description", "dataType", "nullable", "isPartition", "isBucket"))
   expect_equal(collect(c)[[1]][[1]], "speed")
   expect_error(listColumns("zxwtyswklpf", "default"),
-               paste("Error in listColumns : analysis error - Table",
-                     "'zxwtyswklpf' does not exist in database 'default'"))
+               paste("Table or view not found: spark_catalog.default.zxwtyswklpf"))
```

Comment:
This actually is a user behavior change as it returns a different error message now?

Reply:
I don't think we treat error message change as behavior change. We change error messages from time to time.

Reply:
SG